data/korath/korath ships.txt

```diff
@@ -309,89 +309,843 @@ ship "Korath Chaser"
+	turret 77 -116.5 "Shunt-Strike Turret"
+	turret -87 -83 "Korath Banisher"
+	turret 87 -83 "Korath Warder"
+	turret -51 276.5 "Korath Warder"
+	turret 51 276.5 "Korath Banisher"
+	explode "tiny explosion" 30
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 50
+	"final explode" "final explosion large"
+	description "The largest of Korath production vessels are derived from a common and ancient chassis. This bulbous ship has been adapted to carry passengers between stations, or boarding parties into enemy strongholds."
+
+
+ship "Ra'at Ik 621"
+	sprite "ship/world-ship e"
+	thumbnail "thumbnail/world-ship e"
+	attributes
+		category "Heavy Warship"
+		cost 33590000
+		shields 48000
+		hull 31400
+		"required crew" 794
+		"bunks" 1037
+		"mass" 1762
+		"drag" 27
+		"heat dissipation" .41
+		"fuel capacity" 1000
+		"cargo space" 228
+		"outfit space" 999
+		"weapon capacity" 407
+		"engine capacity" 277
+		weapon
+			"blast radius" 400
+			"shield damage" 8000
+			"hull damage" 4000
+			"hit force" 12000
+	outfits
+		"Korath Grab-Strike" 3
+		"Korath Banisher" 2
+		"Korath Warder" 2
+		"Korath Fire-Lance" 3
+		
+		"Triple Plasma Core"
+		"Systems Core (Large)"
+		"Large Heat Shunt" 2
+		"Fuel Processor"
+		"Microbot Defense Station" 16
+		"Korath Repeater Rifle" 308
+		
+		"Reverser (Lunar Class)"
+		"Thruster (Stellar Class)"
+		"Steering (Stellar Class)"
+		"Jump Drive"
+	
+	engine -42 343 .6
+	engine -22 343 .6
+	engine -22 343 .6
+	engine 22 343 .6
+	engine 22 343 .6
+	engine 42 343 .6
+	"reverse engine" -16 -336 0.9
+	"reverse engine" 16 -336 0.9
+	fighter -100 -169
+	fighter 0 -169
+	fighter 100 -169
+	fighter -98 156 back
+	fighter 98 156 back
+	fighter -50 303 back
+	fighter -50 303 back
+	fighter 50 303 back
+	fighter 50 303 back
+	drone -100 -169
+	drone 100 -169
+	drone -98 156 back
+	drone 98 156 back
+	drone -50 303 back
+	drone 50 303 back
+	gun -71 -209 "Korath Fire-Lance"
+	gun 0 -209 "Korath Fire-Lance"
+	gun 71 -209 "Korath Fire-Lance"
+	turret -52 -295 "Korath Banisher"
+	turret 52 -295 "Korath Warder"
+	turret -104 -13 "Korath Grab-Strike"
+	turret 0 -13 "Korath Grab-Strike"
+	turret 104 -13 "Korath Grab-Strike"
+	turret -80.5 191.5 "Korath Warder"
+	turret 80.5 191.5 "Korath Banisher"
+	explode "tiny explosion" 30
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 50
+	"final explode" "final explosion large"
+	description "The largest of Korath production vessels are derived from a common and ancient chassis. This many-lobed ship has been adapted to carry fighters into large-scale raids, or mining craft into asteroid belts. Its radial arrangements of hangars allows fighters a maximum variety of safe approach patterns in combat."
+
+
+ship "Kasichara A'awoj"
+	sprite "ship/world-ship f"
+	thumbnail "thumbnail/world-ship f"
+	attributes
+		category "Space Liner"
```

Comment:
This really seems like it belongs in the Utility category.

Reply:
I rather like the symmetry of two liners, two warships, and two freighters.... and this is the least well-armed of the six..

but, i suppose the crew:cargo ratio is a bit low

could raise crew, reduce cargo.... but, while these will be generally uncapturable, we might want one to show up in a climactic mission or *maybe* two, and it's already a *lot* of bunks even for a trophy